patches/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java.patch

```diff
@@ -0,0 +1,20 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java
+@@ -110,16 +110,7 @@
+ 
+             private float func_178255_b(JsonObject p_178255_1_)
+             {
+-                float f = JsonUtils.func_151217_k(p_178255_1_, "angle");
+-
+-                if (f != 0.0F && MathHelper.func_76135_e(f) != 22.5F && MathHelper.func_76135_e(f) != 45.0F)
```

Comment:
Just adding `false && ...` to the if statement here should work, and makes a smaller patch.